app/bundles/FormBundle/Entity/Form.php

```diff
@@ -601,6 +601,29 @@ public function getFieldAliases()
         return $aliases;
     }
 
+    /**
+     * Loops through the form fields and returns an array of fields with mapped data.
+     *
+     * @return array<string, string>
+     */
+    public function getMappedFieldObjectData(): array
+    {
+        $fields = array_filter(
+            $this->getFields()->map(
+                function (Field $field) {
+                    return [
+                        'mappedObject' => $field->getMappedObject(),
+                        'mappedField'  => $field->getMappedField(),
+                        'fieldAlias'   => $field->getAlias(),
+                    ];
+                }
+            )->toArray(),
+            fn ($elem) => isset($elem['mappedObject'])
+        );
+
+        return array_map('unserialize', array_unique(array_map('serialize', $fields)));
```

Comment:
I used it to remove duplicate values from a multi-dimensional array. I added 'fieldAlias' later 

I changed this code and removed this line because it is not necessary any more.